wit/client.go

```diff
@@ -128,3 +128,31 @@ func GetSpace(ctx context.Context, client *api.Client, spaceID uuid.UUID) (*api.
 	}
 	return client.DecodeSpaceSingle(resp)
 }
+
+func GetSpaces(ctx context.Context, client *api.Client, spaceIDs []uuid.UUID) ([]*api.SpaceSingle, error) {
+	var spaces []*api.SpaceSingle
+	for _, spaceID := range spaceIDs {
+		space, err := GetSpace(ctx, client, spaceID)
```

Comment:
Re-used, GetSpace() which returns one space only.  So for 10 spaceIDs, it will make 10 http call.  Any better alternative available ?

Reply:
I gave this a thought, I don't think we have an alternative. It is *okay* as of now - we should be resilient to this load as of :)

Reply:
We need a Batch GET for Spaces, but not much of an issue here. Unlikely a Codebase would belong to more than one space.